tests/expected/cpp/30721-Issue_2386.cpp

```diff
@@ -0,0 +1,19 @@
+// A
+
+
+std::sting getText()
```

Comment:
std::string, not std::sting, not sure if it matters here or not !?

Reply:
It doesn't matter.